README.md

```diff
@@ -24,15 +24,16 @@ Let this module handle file creation and systemd reloading.
 Or handle file creation yourself and trigger systemd.
 
 ```puppet
-include ::systemd
+include ::systemd::systemctl::daemon_reload
```

Comment:
I see you contained the class in `init.pp`, so I'm guessing the README is not up-to-date?